internal/ptrutil/ptr.go

```diff
@@ -0,0 +1,11 @@
+package ptrutil
```

Comment:
I'm uncertain about this package for a few reasons.

- We've gotten this far without a new package for it.
- `pkg` is usually where we put these kinds of utilities. If we do keep it, we may as well let it be accessible from enterprise too.
- The `Ptr` suffix on the functions stutters with the package name.

Would you mind soliciting some feedback from the team about naming and placement for these, if we keep them?

Reply:
I'm totally cool with moving it to pkg.